core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -207,9 +198,10 @@ export class QuestionBackendApiService {
   }
 
   fetchQuestionSummaries(
-      skillIds: string[], cursor: string): Promise<QuestionSummariesResponse> {
+      skillId: string, cursor?: string): Promise<QuestionSummariesResponse> {
     return new Promise((resolve, reject) => {
-      this._fetchQuestionSummaries(skillIds, cursor, resolve, reject);
+      cursor = cursor ? cursor : '';
```

Comment:
You can remove this if you follow my previous suggestion.

Reply:
done